paper/paper_draft.qmd

```diff
@@ -113,11 +111,13 @@ The distribution of number of years above carbonate-cover thresholds is right-sk
 
 ### Timeseries clustering
 
-To identify groups of reefs at each analysis scale that are resilient, intermediate and non-resilient we clustered reef coral cover timeseries into three clusters at each scale. To quantify the behaviour of timeseries and identify timeseries that behave similarly we calculate the complexity-invariant distance (CID) between each timeseries. The complexity-invariant distance refers to the Euclidean distance between two timeseries multiplied (corrected) by the ratio of the complexities of the timeseries [@batista2014]. These values create a distance matrix of n * n dimensions (where n is the number of timeseries in the group) that is clustered using a k-medoids clustering algorithm [@steinmann2020].
+To identify groups of reefs that are resilient, intermediate, and non-resilient, we clustered reefs by their temporal behaviour as indicated by their coral cover over time. The Complexity-Invariant Distance (CID) between each timeseries was calculated to identify timeseries that behave similarly [@batista2014]. The CID corrects the Euclidean distance between pairs of timeseries (which accounts for magnitude differences) with the complexity of each timeseries (obtained through a measure of their variability) to ensure reefs are clustered based on their underlying temporal dynamics rather than simply their average coral cover. The process creates a distance matrix of n * n dimensions (where n is the number of timeseries in the group) that is clustered using a k-medoids clustering algorithm [cf. @steinmann2020].
```

Comment:
Note I wrote `[cf. @steinmann2020]` at end of paragraph.

This is a slightly old convention, "cf" being the shorthand for the latin _confer_ which means "compare [with]".

I did this because Steinmann did not use k-medoids in their study, but the approach is largely identical.